crescent/ext/cooldowns/__init__.py

```diff
@@ -15,11 +15,22 @@ def _default_bucket(ctx: Context) -> Snowflake:
     return ctx.user.id
 
 
+async def _default_callback(ctx: Context, retry: float) -> None:
+    seconds = round(retry)
+    if seconds == 0:
+        message = "a moment"
```

Comment:
I think its better to lie and say "1 second" since its a clearer time frame than a moment

Reply:
Fair enough